### ISSUE TITLE
Derivative function has mismatched arguments names

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1650]
**Reported by stefanv on 22 Jan 2015 17:12 UTC**
Modelica.Media.Water.IF97_Utilities.T_props_ph has these inputs:

```mo
input Real p;
input Real h;
input Modelica.Media.Common.IF97BaseTwoPhase properties;
```

It's declared derivative, Modelica.Media.Water.IF97_Utilities.T_ph_der, has these inputs:

```mo
input Real p;
input Real h;
input Modelica.Media.Common.IF97BaseTwoPhase aux; // wrong name!
input Real p_der;
input Real h_der;
```

According to ticket#1548, the names of corresponding parameters of a function and its derivative must match. `properties` vs. `aux` violates this, even though `properties` is declared in a `noDerivative` annotation.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1650

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 23 Jan 2015 00:22 UTC**
Thanks for finding this. I think this kind of bug can be fixed for the proposed Milstone, it is a fix.

**Comment by modelica-trac-importer** [comment]
**Comment by kkowalska on 28 Jan 2015 19:55 UTC**
The same problem exists in Modelica.Media.Water.IF97_Utilities.rho_props_ph and Modelica.Media.Water.IF97_Utilities.s_props_ph functions.

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 22 Jun 2015 13:48 UTC**
Fixed in trunk in 14a877a3d926461047d8d3dbacd51cf2360bfb2c

**Comment by modelica-trac-importer** [comment]
**Comment by hubertus on 22 Jun 2015 14:59 UTC**
Fixed in the release branch in 84000601efc8b6900f1b42123930cb5ebe1baf74

**Comment by modelica-trac-importer** [comment]
**Changelog modified by hubertus on 22 Jun 2015 14:59 UTC**
Bug in derivative function arguments in Modelica.Media fixed
